client/src/components/CourseInfo/CourseInfo.tsx

```diff
@@ -61,19 +66,25 @@ const Description = ({ text }: { text: string }) => {
   }, [text, paragraphRef.current]);
```

Comment:
Need to add the window width to dependency array. If you get a course desc that fits on 3 lines (FNCE 449 at largest mobile width should do it) then change the screen size so that it doesn't fit in 3 lines, the more/less button won't render